fission-web-server/library/Fission/Web/Server/Swagger.hs

```diff
@@ -28,6 +28,8 @@ import           Fission.Web.Server.Internal.Orphanage.BasicAuthData ()
 import           Fission.Web.Server.Internal.Orphanage.HigherOrder   ()
 import           Fission.Web.Server.Internal.Orphanage.RegisterDid   ()
 
+import qualified Paths_fission_web_server                            as Fission
```

Comment:
I ran across this the other day — apparently the GHC comes with a bunch of info about the current package available with this weird looking import.

Reply:
Just hopping in here to say: That's a cabal thing: https://www.haskell.org/cabal/release/cabal-2.4.1.0/doc/users-guide/developing-packages.html#accessing-data-files-from-package-code

Reply:
Err, Cabal. Yes. Sorry, was typing past midnight 😛